tests/functions.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+function isPartialRun(): bool
```

Comment:
Could you add a comment above to explain the goal of the file and the function?

Reply:
sure!

Reply:
done